pyproject.toml

```diff
@@ -0,0 +1,45 @@
+[tool.poetry]
+name = "zoloto"
+version = "0.0.0"
+description = "A fiducial marker system powered by OpenCV - Supports ArUco and April"
+authors = ["Jake Howard <git@theorangeone.net>"]
+readme = "README.md"
+repository = "https://github.com/RealOrangeOne/zoloto"
+homepage = "https://github.com/RealOrangeOne/zoloto"
+
+[tool.poetry.dependencies]
```

Comment:
You appear to be missing the optional additional dependency for the rpi.

Reply:
f183f13